routesrv/eskipbytes.go

```diff
@@ -61,11 +65,23 @@ func (e *eskipBytes) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	span := tracing.CreateSpan("serve_routes", r.Context(), e.tracer)
 	defer span.Finish()
 
+	if r.Method != "GET" && r.Method != "HEAD" {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+
 	data, etag, lastModified, initialized := e.bytes()
 	if initialized {
 		w.Header().Add("Etag", etag)
 		w.Header().Add("Content-Type", "text/plain; charset=utf-8")
-		http.ServeContent(w, r, "", lastModified, bytes.NewReader(data))
+		w.Header().Add(routing.RoutesCountName, strconv.Itoa(e.count))
```

Comment:
Here `e.count` is read without holding the mutex. Either `e.bytes()` should be extended to return count or we should drop it and lock/unclock mutex manually

Reply:
done